### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by grantland**
* Remove unnecessary files
* Clean up `ParseQueryAdapter`

### CODE REVIEW

**Comment by grantland** [comment]
We don't actually use our reference to `context` anymore except in `ParseQueryAdapter#getContext()`. We could remove this reference and `@Deprecate` it, but I'm not sure if it's worth the effort.

**Comment by grantland** [comment]
Updated

File: ParseUI-Widget/src/androidTest/java/com/parse/ParseQueryAdapterTest.java [hunk 1]
```diff
@@ -59,13 +61,18 @@ public ParseQueryAdapterTest() {
...
-  private List<ParseObject> savedThings = new ArrayList<ParseObject>();
+  private ListView listView;
+  private List<ParseObject> savedThings;
+  private int totalThings;
 
   @Override
   public void setUp() throws Exception {
     super.setUp();
 
+    listView = new ListView(activity);
```
Comment (wangmengyan95) [inline ParseUI-Widget/src/androidTest/java/com/parse/ParseQueryAdapterTest.java]: :+1:

**Comment by wangmengyan95** [comment]
LGTM!

**Pull request merged**